File: pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.entity.PageResult;
import com.pinyougou.pojo.TbBrand;

import java.util.List;
import java.util.Map;

public interface BrandService {

    public List<TbBrand> findAll();

    public PageResult<TbBrand> findPage(int pageNum, int pageSize);

    public void insert(TbBrand tbBrand);

    public TbBrand findOne(Long id);

    public void update(TbBrand tbBrand);

    public void delete(Long[] lids);

    public PageResult<TbBrand> search(TbBrand tbBrand,int pageNum, int pageSize);

    public List<Map> selectBrandList();
}

File: pinyougou-manager-web/src/main/webapp/js/service/brandService.js
app.service("brandService",function ($http) {
    //分页条件查询
    this.search = function (page, size,searchEntity) {
        return  $http.post("../brand/search.do?page=" + page + "&size=" + size,searchEntity);
    };
    //添加
    this.insert = function (entity) {
        return  $http.post("../brand/insert.do", entity);
    };
    //修改
    this.update = function (entity) {
        return  $http.post("../brand/update.do", entity);
    };
    //显示需要修改的品牌信息
    this.findOne = function (id) {
        return $http.get("../brand/findOne.do?id="+id);
    }
    //删除
    this.dele = function (ids) {
        return $http.get("../brand/del.do?ids="+ids);
    }
    this.selectBrandList = function () {
        return $http.get("../brand/selectBrandList.do");
    }

});